components/Forms/AssigneeEditor.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import AssigneeForm from './AssigneeForm';
+import AssigneeItem from './AssigneeItem';
+import { addUserToProject, deleteUserFromProject, setProjectField } from '../../redux/projects/actions';
+
+
+const AssigneeEditor = ({ projectId, projectUsers }) => {
+  const users = useSelector((store) => store.users.items);
+  const dispatch = useDispatch();
+
+  const userOptions = (projectUsersKey) => {
+    const unusedUsers = users
+      .filter((elem) => !projectUsers
+        .find((prjUser) => elem.id === prjUser.id));
+    const options = unusedUsers.map((user) => (
+      {
+        value: user.id,
+        label: <AssigneeItem email={user.email} name={user.name ? user.name : user.email} />,
+        user,
+        projUserKey: projectUsersKey,
```

Comment:
Variable name `projectUsersKey ` does not work here instead of `projUserKey`?

Reply:
yep. Not any reason for projectUsersKey existing